docs/python-configuration.md

```diff
@@ -4,6 +4,89 @@ title: Scope Python Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified using environment variables or a configuration file `scope.yml` in the source root of the project.
+
+> If these properties are manually configured, they will be `true` only on encountering the string `true` configured on the environment variable. Any other value will be considered as `false`.
+
+These configuration settings can also be provided via CLI flags. Check out the
+[CLI reference](https://scope-python-agent.readthedocs.io/en/latest/cli.html) documentation for details.
+
+## Disabling Scope instrumentation
+
+Scope auto instruments your code and tests by default. You can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=false
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  scope:
+  instrumentation:
+    enabled: false
+
+
+```
+
+
+
+## Changing service name
+
+You can specify the name of the service when sending data to Scope. Scope uses `default` as fallback.
+
+### Using Environment Variables
+
+```sh
+SCOPE_SERVICE="service-name"
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  service: 'service name'
```

Comment:
```suggestion
  service: 'service-name'
```

Reply:
changed all, sorry I thought I had changed all in previous commit, I forgot all that had `'` instead of `"`